### PR TITLE
[TASK] Use decrease cooldown before the nuke

### PR DESCRIPTION
**Pull request by rand0m-user**
Deceases the cooldown of the nuke to 45min

### CODE REVIEW

**Comment by rand0m-user** [comment]
@RockyTV - pass the name to log. e.g. if you want to use the cooldown on another ability you can sue the same function.

File: autoPlay.user.js [hunk 1]
```diff
@@ -926,6 +927,13 @@ function useMetalDetectorIfRelevant() {
 	}
 }
 
+function useDecreaseCooldown(beforeAbility) {
+	if (!isAbilityCoolingDown(ABILITIES.COOLDOWN)) {
+		advLog("Decrease Cooldown before " + beforeAbility + ".", 2);
+		triggerAbility(ABILITIES.COOLDOWN);
```
Comment (iakat) [inline autoPlay.user.js:933]: I think this should be `disableCooldownIfRelevant`, as it checks if cooldown is already active in lane.

Comment (rand0m-user) [inline autoPlay.user.js:933]: both or? i need also to check if it is already purchesd.
Also if i use disableCooldownIfRelevant the UI will toggle, but no boolean for the status.

Comment (rand0m-user) [inline autoPlay.user.js:933]: if we use it for nukes, it is ensured to be active and others also benefit. Right now we have no other usage.

**Comment by rand0m-user** [comment]
please refactor, if something is missing and close this PR

**Comment by iakat** [comment]
This isn't necessary as cooldown is triggered on main loop if not active in lane already.

**Pull request closed without merging**